internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -99,14 +99,6 @@ private ConfigKeys() {
    */
   public static final String KAFKA_LOG_COMPACTION_FOR_HYBRID_STORES = "kafka.log.compaction.for.hybrid.stores";
 
-  /**
-   * Whether to turn on Kafka's log compaction for the store-version topics of incremental push stores.
-   *
-   * Will take effect at topic creation time, and when the incremental push config for the store is turned on.
-   */
-  public static final String KAFKA_LOG_COMPACTION_FOR_INCREMENTAL_PUSH_STORES =
```

Comment:
Is this config not useful at all?

Reply:
Yes. It used to determine whether to turn on LC or not